Packet++/src/TcpReassembly.cpp

```diff
@@ -113,7 +113,7 @@ TcpReassembly::~TcpReassembly()
 	}
 }
 
-void TcpReassembly::reassemblePacket(Packet& tcpData)
+void TcpReassembly::reassemblePacket(Packet& tcpData, ReassemblyStatus &status)

```

Comment:
Is there an initial value we'd like to set?

Reply:
It's a non-const reference so I don't think we can provide a default value...

Reply:
If we change the API as I suggested here: https://github.com/seladb/PcapPlusPlus/pull/435#discussion_r428446802 this won't be necessary